Makefile

```diff
@@ -92,7 +92,7 @@ endif
 test-dev: verify test-browser-dev test-webpack
 
 deploy:
-	webpack --bail --config build/deploy/webpack.config.js
+	webpack --bail --config build/deploy/webpack.deploy.config.js --define process.env.NODE_ENV="'production'"
```

Comment:
Yes, so this setting invokes the webpack `DefinePlugin`. What that does is goes through the code and replaces any instance of `process.env.NODE_ENV` with the string `'production'` so that 
```js
if (process.env.NODE_ENV === 'production') { /* stuff */ }
```
becomes
```js
if ('production' === 'production') { /* stuff */ }
```
which becomes
```js
if (true) { /* stuff */ }
```
and then the uglify step just removes the `if` statement completely to become just 
```
/* stuff */
```